lib/open_telemetry/honeycomb/json/jason_backend.ex

```diff
@@ -0,0 +1,4 @@
+if Code.ensure_loaded?(Jason) do
+  require Protocol
+  Protocol.derive(Jason.Encoder, OpenTelemetry.Honeycomb.Event)
+end
```

Comment:
Took me a while to remember what was going on, here. :) `Jason` implements everything needed to satisfy `OpenTelemetry.Honeycomb.Json` so it's its own `json_module`. There's no need for a module here. That said, we do need the code you put in this module. Would you mind moving these two lines to `Event`?